libs/application/templates/parental-leave/src/fields/Review/index.tsx

```diff
@@ -76,6 +78,22 @@ const Review: FC<FieldBaseProps> = ({ field, application }) => {
   )
 
   const dob = getExpectedDateOfBirth(application)
+  const { data, error, loading } = useQuery(getEstimatedPayments, {
+    variables: {
+      input: {
+        dateOfBirth: dob,
+        period: [
+          {
+            from: '2021-01-01',
+            to: '2021-01-01',
+            ratio: 100,
```

Comment:
Should we remove these (ratio, approved, paid) properties from the input dto ?

Reply:
was going to do that in a later PR